src/chapters/http/web-addresses.rst

```diff
@@ -0,0 +1,123 @@
+Web Addresses
+=============
+
+.. index:: ! URL, ! web address
+
+When a client makes a request to a server, it does so using a
+**uniform resource locator (URL)**. URLs are also called **web addresses**.
+
+.. admonition:: Examples
+
+   As regular users of the web, we are already familiar with URLs like these:
+
+   - ``https://education.launchcode.org/lchs/``
+   - ``https://en.wikipedia.org/wiki/Client–server_model``
+   - ``https://repl.it``
+
+A URL contains a lot of information about the request, including *what* is
+being requested and *where* it should be sent. URLs contain several parts, and
+the general structure looks like:
+
+::
+
+   scheme://host:port/path?query_string
+
+The five parts of this URL are:
+
+#. Scheme
+#. Host
+#. Port (optional)
+#. Path (optional)
+#. Query String (optional)
+
+Only the ``scheme`` and ``host`` are required, and the parts must be in the
+*exact* order shown. Let's look at each of these in detail.
+
+Scheme
+------
+
+.. index::
+   single: url; scheme
+
+The first portion of every URL specifies the **scheme**. Common schemes include
+``http``, ``https``, and ``file``. The scheme usually identifies the *protocol*
+for the request. For us, this will always be ``http`` or ``https``. If left
+off, a web browser will insert the scheme http/s for you. 
+
+The scheme is *always* followed by ``://``.
+
+Host
+----
+
+.. index::
+   single: url; domain
+
+The **host** portion of a URL specifies where the request should be sent. The
+host can be either an IP address, like ``104.25.128.113``, or a domain name,
+like ``www.launchcode.org``.
+
+Port
+----
+
+.. index::
+   single: url; port
+
+Following the host is an optional **port** number. The port determines which
+application on the server needs to handle the request.
+
+When the port number is missing, a default value is used. For ``http://``, the
+default port is ``80``. When using ``https://``, the default port is ``443``.
```

Comment:
An additional sentence could help here: "On the server, port 80 and 443 would specify to the server computer that the request should be handled by a specific program on that server that can handle web requests." (or something like this...)

Reply:
I updated this by adding a follow-up paragraph.  Let me know what you think.